src/createBaseForm.js

```diff
@@ -270,9 +276,6 @@ function createBaseForm(option = {}, mixins = []) {
           inputProps[fieldDataProp] = this.fieldsStore.getField(name);
         }
 
-        // This field is rendered, record it
-        this.renderFields[name] = true;
```

Comment:
Hmmm....I missed here. These code should keep cause user can call `getFieldDecorator('field')` to keep a field in store and not render it.
I guess your prev new variable solution is better. My bad.

Reply:
Fine, I'll revert the commit.

Reply:
@zombieJ Update PR